### DISCUSSION TITLE
windowpos

### DISCUSSION THREAD

**Discussion by alverman** [sumatrapdfreader/sumatrapdf#3482] [Q&A]
Good morning,
every time you move sumatra from one monitor to another the position is saved.
Is it possible to disable this saving.
I would need the position value to always remain the same, otherwise I would have to open the file from code and modify it each time before opening sumatra.

Thank you, Alberto

**Comment by GitHubRulesOK** [comment]
Hmm the idea is it re-opens where last placed (its often that way with many apps)
so as you say you would need to edit settings to "reset" it to a fixed location.

One way would be to make the file RO (read only) with a fixed offset (top left is roughly `-4 0 width height`) however that means all other settings will be frozen and for a second screen need an unknown offset. 

Another way is to place the desired location at top of settings before opening and it should trickle down into position HOWEVER beware windows BOM can cause overwrite issues, a simpler method would be find and replace the whole line as plain text regardless of last value.

**Reply by alverman** [reply]
Thanks for advice

**Reply by GitHubRulesOK** [reply]
If you have a suitable temporary TRIMMED DOWN file like the upper one as your default (I used 1024x768 )

and place that (overwrite by copy) just before  open SumatraPDF it will write all the other defaults into the rest of file and move those accepted entries down to normal location. (this avoids writing a routine other than copy settings before start but naturally there will be NO file actions remembered from previous session changed settings.
![image](https://user-images.githubusercontent.com/12754913/236540837-990ddd85-bad0-45ea-a55b-5e4572c7b608.png)
